### ISSUE TITLE
2019-12-18 tc113 M3SD doesn't generate CMINDN

### ISSUE THREAD

**Issue by tensfeldt** [tensfeldt/openNCA#124]
With commit 560bcad, tc113 M3SD doesn't generate CMINDN with parameterset=="PARAMETERLIST". An updated MCT, MCT113.json/.csv was provided to onedrive including all parameters expected for M3SD for tc113.

Note that CMINDN isn't currently defined in the Computation Engine specifications document. It will be added to the version in the master branch.

```r
> chkresults(dataset=d, mct=mct, parameterset=parameterset, results=r)
parameterset:  PARAMETERLIST 
[1] "AUCDN"    "AUCT"     "AUCT1_T2" "AUCTDN"   "CMINDN"  
```

```r
> r
      SDEID DOSE   DOSEC  DOF CENDINF CENDINFDN TENDINF  CMAX FLGACCEPTPREDOSE CMIN CLAST     CEST   CMAXDN     TMAX FLGACCEPTTMAX TMIN    TLAST        KEL      KELC0  KELTMLO  KELTMHI KELNOPT      KELR
1  49420991  100 1.0e+08 1.00      NA        NA      NA 30900                1    0  11.1 10.59422 309.0000 1.500000             1    0 96.83333 0.08081319 26521.6973 13.00000 96.83333       5 0.9998584
2 157748714   50 5.0e+07 1.00    5430     108.6       1  6550                1    0  12.5 15.41977 131.0000 1.500000             1    0 73.00000 0.08291236  6556.7659 13.00000 73.00000       4 0.9926834
3 216997173   15 1.5e+07 1.00      NA        NA      NA  3640                1    0  49.8 39.94352 242.6667 1.500000             1    0 48.91667 0.05827129   690.8368 13.00000 48.91667       3 0.8761725
4 281691464  200 2.0e+08 1.02      NA        NA      NA 69300                1    0  36.8 35.34179 346.5000 1.016667             1    0 73.01667 0.09627129 39913.6987 13.01667 73.01667       4 0.9997891
5 293032605  200 2.0e+08 1.02      NA        NA      NA 33400                1    0  10.5 10.40005 167.0000 1.033333             1    0 48.95000 0.15124119 17069.8160 13.01667 48.95000       3 0.9999579
     KELRSQ   KELRSQA FLGACCEPTKEL     THALF THALFF LASTTIME    AUCALL  AUCALLDN   AUCLAST  AUCLASTC AUCLASTDN  AUMCLAST AUC1      AUC2     AUC3       AUC4      AUC5      AUC6      AUC7     AUC8      AUC9
1 0.9997168 0.9996225            1  8.577154      0 504.9167 330294.65 3302.9465 330161.45 330161.45 3301.6145 4163056.1 3050  7777.500 16719.17  42973.828  76794.34 115030.90 163615.46 215395.3 287789.20
2 0.9854203 0.9781304            1  8.359998      0 505.0000  69080.76 1381.6152  68930.76  68930.76 1378.6152  952784.1  805  2967.500  5962.50  11031.450  17064.22  23391.19  31593.95  41237.0  57047.68
3 0.7676783 0.5353565            0 11.895176      0 505.0000  29683.01 1978.8673  29085.41  29085.41 1939.0273  230013.6  241  1252.117  2844.70   6337.694  10865.75  15948.61  21343.29  24736.2  27523.54
4 0.9995782 0.9993673            1  7.199936      0 505.0167 475729.12 2378.6456 475287.52 475287.52 2376.4376 4518255.3 6200 30509.167 60737.58 110647.499 173285.72 234629.63 298719.23 356230.8 438678.50
5 0.9999158 0.9998315            1  4.583058      0 505.0167 170313.66  851.5683 170187.66 170187.66  850.9383  948976.5 4500 18206.667 33298.03  60511.984  90189.12 116041.85 139204.05 154515.1 167711.76
      AUC10     AUC11     AUC12     AUC13     AUC14     AUC15     AUC16     AUC1DN     AUC2DN    AUC3DN    AUC4DN    AUC5DN    AUC6DN    AUC7DN    AUC8DN    AUC9DN   AUC10DN   AUC11DN   AUC12DN   AUC13DN
1 324146.13 329388.18 330161.45 330294.65 330294.65 330294.65 330294.65 0.03050000 0.07777500 0.1671917 0.4297383 0.7679434 1.1503090 1.6361546 2.1539527 2.8778920 3.2414613 3.2938818 3.3016145 3.3029465
2  67590.57  68930.76  69080.76  69080.76  69080.76  69080.76  69080.76 0.01610000 0.05935000 0.1192500 0.2206290 0.3412845 0.4678238 0.6318789 0.8247399 1.1409536 1.3518114 1.3786152 1.3816152 1.3816152
3  29085.41  29683.01  29683.01  29683.01  29683.01  29683.01  29683.01 0.01606667 0.08347444 0.1896467 0.4225129 0.7243830 1.0632404 1.4228863 1.6490799 1.8349029 1.9390273 1.9788673 1.9788673 1.9788673
4 472079.64 475287.52 475729.12 475729.12 475729.12 475729.12 475729.12 0.03100000 0.15254583 0.3036879 0.5532375 0.8664286 1.1731481 1.4935962 1.7811540 2.1933925 2.3603982 2.3764376 2.3786456 2.3786456
5 170187.66 170313.66 170313.66 170313.66 170313.66 170313.66 170313.66 0.02250000 0.09103333 0.1664902 0.3025599 0.4509456 0.5802092 0.6960203 0.7725755 0.8385588 0.8509383 0.8515683 0.8515683 0.8515683
    AUC14DN   AUC15DN   AUC16DN AUCINT1            AUCINT2            AUCINT3            AUCINT4            AUCINT5            AUCINT6            AUCINT7            AUCINT8            AUCINT9
1 3.3029465 3.3029465 3.3029465   0_0.5 0_1.01666666666667              0_1.5              0_2.5                0_4                0_6                0_9               0_13 0_24.9166666666667
2 1.3816152 1.3816152 1.3816152   0_0.5                0_1              0_1.5              0_2.5                0_4                0_6                0_9               0_13               0_25
3 1.9788673 1.9788673 1.9788673   0_0.5 0_1.01666666666667              0_1.5              0_2.5                0_4                0_6                0_9               0_13 0_24.9166666666667
4 2.3786456 2.3786456 2.3786456   0_0.5 0_1.01666666666667 0_1.51666666666667 0_2.51666666666667 0_4.01666666666667 0_6.01666666666667 0_9.01666666666667 0_13.0166666666667 0_25.0166666666667
5 0.8515683 0.8515683 0.8515683   0_0.5 0_1.03333333333333              0_1.5              0_2.5 0_4.01666666666667 0_6.01666666666667 0_9.01666666666667 0_13.0166666666667            0_24.95
            AUCINT10           AUCINT11           AUCINT12           AUCINT13           AUCINT14           AUCINT15           AUCINT16  AUC0_336  AUC0_168   AUCINFO   AUCINFP  AUCINFOC  AUCINFPC AUCINFODN
1 0_48.8333333333333 0_72.8333333333333 0_96.8333333333333 0_120.833333333333 0_168.833333333333 0_336.916666666667 0_504.916666666667 330294.65 330295.45 330298.81 330292.55 330298.81 330292.55 3302.9881
2               0_49               0_73               0_97              0_121              0_169 0_337.033333333333              0_505  69080.76  69080.90  69081.52  69116.74  69081.52  69116.74 1381.6305
3 0_48.9166666666667 0_72.9166666666667 0_96.9166666666667 0_120.916666666667 0_168.916666666667              0_337              0_505  29683.01  29683.92  29940.03  29770.88  29940.03  29770.88 1996.0022
4 0_49.0166666666667 0_73.0166666666667 0_97.0166666666667 0_121.016666666667 0_169.016666666667 0_337.016666666667 0_505.016666666667 475729.12 475729.21 475669.78 475654.63 475669.78 475654.63 2378.3489
5            0_48.95            0_72.95            0_96.95           0_120.95              0_169 0_337.016666666667 0_505.016666666667 170313.66 170313.66 170257.09 170256.43 170257.09 170256.43  851.2854
  AUCINFPDN  AUMCINFO  AUMCINFP   MRTLAST  MRTIVIFO  MRTIVIFP   AUCXPCTO   AUCXPCTP   AUMCXPTO   AUMCXPTP       CLO      CLOW       CLP      CLPW      VZO       VZOW      VZP       VZPW     VSSO     VSSP
1 3302.9255 4178056.2 4177372.7 12.109152 12.149323 12.147493  0.3590204  0.3427173  0.3590204  0.3427173  302.7562  3.973178  302.7619  3.973253 3.746371 0.04916497 3.746442 0.04916590 3.678282 3.677798
2 1382.3348  965608.0  968603.4 13.322335 13.477804 13.514021  1.3280671  1.6332129  1.3280671  1.6332129  723.7825  7.438669  723.4138  7.434879 8.729489 0.08971726 8.725041 0.08967155 9.754999 9.776229
3 1984.7256  286485.2  275308.3  7.408212  9.068633  8.747568 19.7118767 16.4523493 19.7118767 16.4523493  501.0015  6.574822  503.8480  6.612178 8.597741 0.11283125 8.646591 0.11347232 4.543399 4.407444
4 2378.2731 4550136.8 4548873.4  8.996362  9.055747  9.053396  0.7006697  0.6730923  0.7006697  0.6730923  420.4598  5.618115  420.4731  5.618294 4.367447 0.05835712 4.367586 0.05835898 3.807577 3.806710
5  851.2821  952833.9  952797.2  5.066059  5.086442  5.086248  0.4048364  0.4009983  0.4048364  0.4009983 1174.6941 12.819973 1174.6987 12.820023 7.767025 0.08476509 7.767055 0.08476542 5.975014 5.974809
       VSSOW      VSSPW CONC1 CONC2 CONC3 CONC4 CONC5 CONC6 CONC7 CONC8 CONC9 CONC10 CONC11 CONC12 CONC13 CONC14 CONC15 CONC16 CONC17 CONCTIME1 CONCTIME2 CONCTIME3 CONCTIME4 CONCTIME5 CONCTIME6 CONCTIME7
1 0.04827142 0.04826507     0 12200  6100 30900 22100 23000 15700 16700  9800 3440.0  496.0   70.8   11.1      0      0      0      0         0       0.5  1.016667  1.500000  2.500000  4.000000  6.000000
2 0.10025693 0.10047512     0  3220  5430  6550  3830  4220  2300  3220  1750  963.0  152.0   12.5    0.0      0      0      0      0         0       0.5  1.000000  1.500000  2.500000  4.000000  6.000000
3 0.05962465 0.05784048     0   964  2950  3640  3350  2710  2380  1320   505   83.2   49.8    0.0    0.0      0      0      0      0         0       0.5  1.016667  1.500000  2.500000  4.000000  6.000000
4 0.05087623 0.05086464     0 24800 69300 52400 47500 36500 25500 17700 11500 3690.0  330.0   36.8    0.0      0      0      0      0         0       0.5  1.016667  1.516667  2.516667  4.016667  6.016667
5 0.06520805 0.06520582     0 18000 33400 31300 23500 16100 10200  5680  2430  381.0   10.5    0.0    0.0      0      0      0      0         0       0.5  1.033333  1.500000  2.500000  4.016667  6.016667
  CONCTIME8 CONCTIME9 CONCTIME10 CONCTIME11 CONCTIME12 CONCTIME13 CONCTIME14 CONCTIME15 CONCTIME16 CONCTIME17 TIMEU AMOUNTU DOSEU VOLUMEU CONCU KELU   CLU     AUCU       AUMCU    AUCNORMU AURCU CONCNORMU
1  9.000000  13.00000   24.91667   48.83333   72.83333   96.83333   120.8333   168.8333   336.9167   504.9167    HR      NA    MG       L NG/ML 1/HR ML/HR NG.HR/ML NG.HR.HR/ML NG.HR/ML/MG    NA  NG/ML/MG
2  9.000000  13.00000   25.00000   49.00000   73.00000   97.00000   121.0000   169.0000   337.0333   505.0000    HR      NA    MG       L NG/ML 1/HR ML/HR NG.HR/ML NG.HR.HR/ML NG.HR/ML/MG    NA  NG/ML/MG
3  9.000000  13.00000   24.91667   48.91667   72.91667   96.91667   120.9167   168.9167   337.0000   505.0000    HR      NA    MG       L NG/ML 1/HR ML/HR NG.HR/ML NG.HR.HR/ML NG.HR/ML/MG    NA  NG/ML/MG
4  9.016667  13.01667   25.01667   49.01667   73.01667   97.01667   121.0167   169.0167   337.0167   505.0167    HR      NA    MG       L NG/ML 1/HR ML/HR NG.HR/ML NG.HR.HR/ML NG.HR/ML/MG    NA  NG/ML/MG
5  9.016667  13.01667   24.95000   48.95000   72.95000   96.95000   120.9500   169.0000   337.0167   505.0167    HR      NA    MG       L NG/ML 1/HR ML/HR NG.HR/ML NG.HR.HR/ML NG.HR/ML/MG    NA  NG/ML/MG
  RATEU VOLUMEWU     CLWU    STUDY SITEID   SUBJID RAND           TREATXT TRTCD PKCOLL PKBDFLD                 PKTERM VISITU VISIT             PCANMETH     HT    WT AGEDERU AGEDER WTUNI WTRAW HTUNI HTRAW
1    NA     L/KG ML/HR/LB B2901001   1001 10011043   62 100mg iv infusion     G  POINT  PLASMA PF-05231023 C-TERMINAL    DAY     1 bioanlaytical method 168.91 76.20      YR     50    LB 168.0    IN  66.5
2    NA     L/KG ML/HR/LB B2901001   1005 10051036   52  50mg iv infusion     F  POINT  PLASMA PF-05231023 C-TERMINAL    DAY     1 bioanlaytical method 175.26 97.30      YR     39    LB 214.5    IN  69.0
3    NA     L/KG ML/HR/LB B2901001   1001 10011026   44  15mg iv infusion     E  POINT  PLASMA PF-05231023 C-TERMINAL    DAY     1 bioanlaytical method 177.04 76.20      YR     61    LB 168.0    IN  69.7
4    NA     L/KG ML/HR/LB B2901001   1002 10021073   80 200mg iv infusion     H  POINT  PLASMA PF-05231023 C-TERMINAL    DAY     1 bioanlaytical method 149.86 74.84      YR     56    LB 165.0    IN  59.0
5    NA     L/KG ML/HR/LB B2901001   1003 10031045   78 200mg iv infusion     H  POINT  PLASMA PF-05231023 C-TERMINAL    DAY     1 bioanlaytical method 172.72 91.63      YR     63    LB 202.0    IN  68.0
  RACES SEX          ETHNIC             ACTTRT ACTTRTC ACTTRTS TREATSEQ  PCNAM PCLLOQ PCSTRESU ROUTE FLGEMESIS              DOSEP DOSPND DATASTATUS DOFU
1     1   M HISPANIC/LATINO PF-05231023 100 mg       G       G        G vendor     10    NG/ML    IV         0 PF-05231023 100 mg   DONE      DRAFT   HR
2     1   M HISPANIC/LATINO  PF-05231023 50 mg       F       F        F vendor     10    NG/ML    IV         0  PF-05231023 50 mg   DONE      DRAFT   HR
3     2   M HISPANIC/LATINO  PF-05231023 15 mg       E       E        E vendor     10    NG/ML    IV         0  PF-05231023 15 mg   DONE      DRAFT   HR
4     1   F HISPANIC/LATINO PF-05231023 200 mg       H       H        H vendor     10    NG/ML    IV         0 PF-05231023 200 mg   DONE      DRAFT   HR
5     1   M HISPANIC/LATINO PF-05231023 200 mg       H       H        H vendor     10    NG/ML    IV         0 PF-05231023 200 mg   DONE      DRAFT   HR
 [ reached 'max' / getOption("max.print") -- omitted 45 rows ]
```

**Comment by jhhughes256** [comment]
This still occurs with commit ed0aed3.

**Comment by jhhughes256** [comment]
CMINDN has been implemented as of cca452d. Closing this issue.
